lib/Yancy/Util.pm

```diff
@@ -27,7 +27,8 @@ use Mojo::Base '-strict';
 use Exporter 'import';
 use Mojo::Loader qw( load_class );
 use Scalar::Util qw( blessed );
-our @EXPORT_OK = qw( load_backend curry currym );
+use Mojo::JSON::Pointer;
```

Comment:
Can this commit be moved before https://github.com/preaction/Yancy/pull/59/commits/d4a71ffdfa7628b324e32bb15897d0991d5dc953? Right now, that commit uses this sub, so the commits between are broken.

Reply:
This is GH using the dates of the commits to sort, rather than the actual "parent" chain of the commits. The commits are not in fact broken because I created this new solution, `copy_inline_refs` to the old problem, inserted that new solution *before* the one that uses it, then changed the using one to use the new solution.

I know this is true because I have just finished running `git rebase -x 'TEST_YANCY_EXAMPLES=1 ./xt/run_backend_tests.pl' -i master` in order to be 100% sure of this (which at 1.5 mins * 41 commits takes over an hour). `rebase` gets the order correct, and there is no breakage - every single commit passes every single test :-)

Reply:
Sorry, my bad. Yeah, I got real confused when Github showed me code using that sub before it existed, like "what the heck is this?"

Reply:
This is sadly not a new problem with GH ;-)